nearcore/src/config.rs

```diff
@@ -1272,23 +1274,28 @@ pub fn init_testnet_configs(
         archive,
         tracked_shards,
     );
+    let log_config = LogConfig::default();
     for i in 0..(num_validator_seats + num_non_validator_seats) as usize {
+        let config = &configs[i];
         let node_dir = dir.join(format!("{}{}", prefix, i));
         fs::create_dir_all(node_dir.clone()).expect("Failed to create directory");
 
         validator_signers[i]
-            .write_to_file(&node_dir.join(&configs[i].validator_key_file))
+            .write_to_file(&node_dir.join(&config.validator_key_file))
             .expect("Error writing validator key file");
         network_signers[i]
-            .write_to_file(&node_dir.join(&configs[i].node_key_file))
+            .write_to_file(&node_dir.join(&config.node_key_file))
             .expect("Error writing key file");
         for key in &shard_keys {
             key.write_to_file(&node_dir.join(format!("{}_key.json", key.account_id)))
                 .expect("Error writing shard file");
         }
 
-        genesis.to_file(&node_dir.join(&configs[i].genesis_file));
-        configs[i].write_to_file(&node_dir.join(CONFIG_FILENAME)).expect("Error writing config");
+        genesis.to_file(&node_dir.join(&config.genesis_file));
+        config.write_to_file(&node_dir.join(CONFIG_FILENAME)).expect("Error writing config");
+        log_config
```

Comment:
The message that `log_config.json` is missing is FYI, and doesn't indicate an unexpected condition. Not sure if this needs fixing.
But I guess it upsets you, and therefore LGTM.